dali/test/python/test_plugin_manager.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nvidia.dali.pipeline import Pipeline
+import nvidia.dali.ops as ops
+import nvidia.dali.types as types
+import nvidia.dali.plugin_manager as plugin_manager
+import unittest
+
+image_dir = "../docs/examples/images"
```

Comment:
This path is not relative to this source file, but to "test" directory. I'm not familiar with Python tests - are they supposed to be run from there? If it's not clear that they are, we may have to somehow locate this .py file and use path relative to it.